packages/builders/src/components/Assertions.ts

```diff
@@ -8,40 +8,46 @@ export const customIdValidator = s.string
 	.lengthLessThanOrEqual(100)
 	.setValidationEnabled(isValidationEnabled);
 
-export const emojiValidator = s
+const jsonEmojiValidator = s
 	.object({
 		id: s.string,
 		name: s.string,
 		animated: s.boolean,
 	})
 	.partial.strict.setValidationEnabled(isValidationEnabled);
 
+export const emojiValidator = jsonEmojiValidator.nullable;
+
 export const disabledValidator = s.boolean;
 
 export const buttonLabelValidator = s.string
 	.lengthGreaterThanOrEqual(1)
 	.lengthLessThanOrEqual(80)
-	.setValidationEnabled(isValidationEnabled);
+	.nullable.setValidationEnabled(isValidationEnabled);
 
 export const buttonStyleValidator = s.nativeEnum(ButtonStyle);
 
-export const placeholderValidator = s.string.lengthLessThanOrEqual(150).setValidationEnabled(isValidationEnabled);
+export const placeholderValidator = s.string
+	.lengthLessThanOrEqual(150)
+	.nullable.setValidationEnabled(isValidationEnabled);
 export const minMaxValidator = s.number.int
 	.greaterThanOrEqual(0)
 	.lessThanOrEqual(25)
-	.setValidationEnabled(isValidationEnabled);
+	.nullable.setValidationEnabled(isValidationEnabled);
 
-export const labelValueDescriptionValidator = s.string
+export const labelValueValidator = s.string
```

Comment:
This change affects `builders.ComponentAssertions.labelValueDescriptionValidator`, which stops existing after this PR, and that makes the PR semver-major.

Please restore the name by renaming the export back to `labelValueDescriptionValidator` or re-exporting it with `export { labelValueValidator as labelValueDescriptionValidator };`.